core/lib/server/pub.ts

```diff
@@ -1407,11 +1596,37 @@ export async function getPubsWithRelatedValuesAndChildren<
 					.selectFrom("pubs")
 					.selectAll("pubs")
 					.where("pubs.communityId", "=", props.communityId)
+					.leftJoin("PubsInStages", "pubs.id", "PubsInStages.pubId")
+					.select("PubsInStages.stageId")
+					.where((eb) =>
```

Comment:
this condition should be the same as the `where` above in the recursive part of the query